ManatiCR/vitaminau
# Vit-54: VitaminaU header.

# Task [VIT-54](https://manati.atlassian.net/browse/VIT-54)
Create a custom block for the vitaminaU header.

# Installation
- `ahoy composer install`
- `ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the site at the following URL:
  - http://vitaminau.docker/
- See that the header looks exactly like the jira ticket [VIT-54](https://manati.atlassian.net/browse/VIT-54).
- Check that the menu links direct you to the desired location (at the moment they all work except "Mis borradores").

# CA
- The header should look something like this:
  - Mobile:
![afb05938-4bce-4f78-bd50-3344592a2c63](https://user-images.githubusercontent.com/48957125/93381804-7098f080-f81e-11ea-9674-8cfd085cbc07.png)
  - Desktop:
![e39ea5f3-933f-4c46-b0d7-5fd24db92035](https://user-images.githubusercontent.com/48957125/93381810-72fb4a80-f81e-11ea-9ca7-509b793e0c88.png)
- Menu functionality must be correct.
